images/wkdev_sdk/user_home_directory_defaults/dot-gdbinit

```diff
@@ -6,3 +6,4 @@ set print array-indexes on
 set python print-stack full
 set debuginfod enabled on
 set sysroot /
+handle signal SIGUSR1 nostop noprint
```

Comment:
Oops, didn't notice it before, there's a stray `signal` in there. The line should read `handle SIGUSR1 nostop noprint` (copy-pasted from GDB to avoid any typos).